### PR TITLE
docs(platform): update the readme page in the beginning - typo fixes

### PR DESCRIPTION
**Pull request by ynnenu**
3. Documentation and Example validations

-   [x] missing API documentation or it is not understandable

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -149,6 +149,7 @@
     "rxjs": "7.5.6",
     "tslib": "2.3.0",
     "webdriver-manager": "12.1.7",
+    "yarn": "1.22.19",
```
Comment (platon-rov) [inline package.json:152]: probably you installed this package locally instead of globally, so now we have changes in `yarn.lock` also but we shouldn't

**Review by platon-rov — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (platon-rov) [reply package.json]: reverted

**Review by g-cheishvili — APPROVED** [review]

File: PLATFORM_README.md [hunk 2]
```diff
@@ -40,15 +40,15 @@ This `Platform UI library` is not going to be only about UI but it needs to also
 
 This example captures several things:
 
-a). How we abstract from assembly that me as developer I don't deal with layouting of elements for labels, contrals, hints, etc..
+a). Abstract from assembly that doesn't require any element layout for labels, contrals, hints, etc..
```
Comment (mikerodonnell89) [inline PLATFORM_README.md:43]:
```suggestion
a). Abstract from assembly that doesn't require any element layout for labels, controls, hints, etc..
```

File: PLATFORM_README.md [hunk 3]
```diff
@@ -40,15 +40,15 @@ This `Platform UI library` is not going to be only about UI but it needs to also
...
 
-b). Assembly like this gives pretty big space in terms of different layouts that we want to support in the application (1 column, 2 columns)
+b). Support different layouts in the application (e.g., 1 column, 2 columns).
 
-c). If we need to group information into sections
+c). Group information into sections
 
-d). How we are handling and forms errors
+d). Handle forms errors
```
Comment (mikerodonnell89) [inline PLATFORM_README.md:49]:
```suggestion
d). Handle form errors
```

**Review by mikerodonnell89 — APPROVED** [review]
Pushed some small tweaks

**Pull request merged**